### PR TITLE
Port: less

### PR DESCRIPTION
**Pull request by christopherdumas**
This is a very simple port of less. I needed it as part of solving #136 (working on it at #140), and thought I'd make a PR for it as well.

### CODE REVIEW

File: Ports/.port_include.sh [hunk 1]
```diff
@@ -58,6 +58,14 @@ function run_export_env() {
     export $1="$2"
 }
 
+function run_fetch_wget() {
+	run_command_nocd rm -rf "$PORT_DIR"
```
Comment (awesomekling) [inline Ports/.port_include.sh]: Let's use 4 spaces for indentation in bash scripts as well.

File: Ports/.port_include.sh [hunk 2]
```diff
@@ -58,6 +58,14 @@ function run_export_env() {
     export $1="$2"
 }
 
+function run_fetch_wget() {
```
Comment (awesomekling) [inline Ports/.port_include.sh]: What's the difference between this and run_fetch_web()?

Comment (christopherdumas) [reply Ports/.port_include.sh]: wow I didn't know that was there!

**Comment by christopherdumas** [comment]
Yeah this is just here because it uses ncurses to do scrolling, while our implementation of more (while it is perfectly servicable) does not.

Speaking of which, I'm not stopping until the Terminal is able to do Vim correctly, but that might be awhile.

**Comment by awesomekling** [comment]
That's great to hear, Terminal could use some love :)

**Pull request merged**